api/tests/unit/domain/usecases/get-certification-candidate-subscription_test.js

```diff
@@ -12,176 +13,209 @@ describe('Unit | UseCase | get-certification-candidate-subscription', function (
     certificationCandidateRepository = {
       getWithComplementaryCertifications: sinon.stub(),
     };
+
+    certificationCenterRepository = {
+      getBySessionId: sinon.stub(),
+    };
   });
 
-  context('when the candidate is registered and eligible to one complementary certification', function () {
-    it('should return the candidate with one eligible complementary certification', async function () {
-      // given
-      const certificationCandidateId = 123;
-      const userId = 456;
-      const sessionId = 789;
-
-      const complementaryCertification = domainBuilder.buildComplementaryCertification({ key: 'PIX+' });
-
-      const certifiableBadgeAcquisition = domainBuilder.buildCertifiableBadgeAcquisition({
-        badge: domainBuilder.buildBadge({
-          key: 'PIX+_BADGE',
-          isCertifiable: true,
-        }),
-        complementaryCertification,
-      });
+  context('when certification center is not habilitated', function () {
+    context('when the candidate is registered and eligible to one complementary certification', function () {
+      it('should return the candidate with no eligible complementary certification', async function () {
```

Comment:
En lisant le 'it', il semble qu'on va avoir une ligne dans les nonEligible, alors qu'il doit etre vide. 

```suggestion
      it('should return the candidate without complementary certification', async function () {
```